cfgov/jinja2/v1/_includes/macros/util/format/datetime.html

```diff
@@ -47,9 +47,9 @@
 
 {% macro format_date(datetime, text_format=False) %}
```

Comment:
If the default arg to `text_format` is already `False`, do we really need to pass it around here?